src/hooks/useGetQueryParamOrReduxValue.ts

```diff
@@ -201,22 +199,30 @@ const useGetQueryParamOrReduxValue = (
   }
   // @ts-ignore
   const reduxValue = useSelector(...useSelectorArguments);
-  // TODO: this bit is identical to the one in useGetQueryParamOrXstateValue.ts, keep it DRY
+  const surahReduxValue = useSelector(
+    QUERY_PARAMS_DATA[QueryParam.SURAH].reduxSelector,
+    QUERY_PARAMS_DATA[QueryParam.SURAH].reduxEqualityFunction,
+  );
```

Comment:
This check should not be a general one that applies to every query param but rather it's specific to when it's `from` or `to` query params.

Reply:
This is true, but as you know we can't use hooks conditionally, in our case `useSelector`.

Reply:
Why cannot we make `reduxValue` select whatever `surahReduxValue` is trying to select when there is a `customHandler`?

Reply:
We need both the redux value, which will be the ayah, and the surah redux value simultaneously.